### PR TITLE
Parse Maine government vaccination site list.

### LINKED ISSUE #128: parse maine_gov from Maine
[learn our pipeline: parse](https://github.com/CAVaccineInventory/vaccine-feed-ingest/wiki/Runner-pipeline-stages#parse)

Blocked by #127

Parse existing data into [`.ndjson`](http://ndjson.org/).

Read all files in the directory passed as the second argument (`sys.argv[2]`), convert them to `.ndjson`, and then output them to new files in the directory passed as the first argument (`sys.argv[1]`).

Check the wiki to learn more about the purpose of the parse stage and how to get set up for development!

### Tips

1. Fetch data for this site before you start developing:
    ```sh
    poetry run vaccine-feed-ingest fetch me/main_gov
    ```

1. While working on your code, run it at any point:
    ```sh
    poetry run vaccine-feed-ingest parse me/main_gov
    ```

### Example
[Fetched files for `md/arcgis`](https://github.com/CAVaccineInventory/vaccine-feed-ingest-results/tree/main/md/arcgis/raw) are converted to [parsed files](https://github.com/CAVaccineInventory/vaccine-feed-ingest-results/tree/main/md/arcgis/parsed)

### PR DESCRIPTION
**Pull request by cdjj**
Resolves #128 .

### CODE REVIEW

File: vaccine_feed_ingest/runners/me/maine_gov/parser.py [hunk 1]
```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
```
Comment (eliblock) [inline vaccine_feed_ingest/runners/me/maine_gov/parser.py:1]: Please name this file `parse.py` so it gets picked up by the cli!

File: vaccine_feed_ingest/runners/me/maine_gov/parser.py [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    soup = BeautifulSoup(content, "html.parser")
+
+    healthcenters = soup.find(id="vaccsites")
+    sites.extend(extract_healthcenters(healthcenters))
+
+    pharmacies = soup.find(id="vaccsites2")
+    sites.append(extract_pharmacies(pharmacies))
+
+    mobile = soup.find(id="vaccsites3")
+    sites.append(extract_pharmacies(mobile))
```
Comment (eliblock) [inline vaccine_feed_ingest/runners/me/maine_gov/parser.py:112]: Thank you for handling the various types of data on the page!

**Review by eliblock — CHANGES_REQUESTED** [review]
Hi @cdjj - thank you for adding this! It looks like at the moment the parser does not get picked up by the cli command (`poetry run vaccine-feed-ingest all-stages me/maine_gov`) due to the filename. Once it does, this will be good to go!

To satisfy the linter, please run `poetry run black vaccine_feed_ingest`.

File: vaccine_feed_ingest/runners/me/maine_gov/parser.py (hunk 1 above)
Comment (cdjj) [reply vaccine_feed_ingest/runners/me/maine_gov/parser.py]: Doh, muscle memory!  Fixed.

**Review by eliblock — APPROVED** [review]
Thank you! One day (hopefully soon) we plan to add more meaningful CI which would have given you that feedback directly.

**Comment by eliblock** [comment]
Ah, the linter :) . @cdjj could you run `poetry run black vaccine_feed_ingest`?

**Comment by cdjj** [comment]
Looks like the BeautifulSoup dependency hasn't landed from PR #166 yet, so adding that here.

**Pull request merged**